cogs/auctions.py

```diff
@@ -162,7 +162,7 @@ async def auction(self, ctx):
         await ctx.send_help(ctx.command)
 
     @commands.has_permissions(administrator=True)
-    @auction.command()
+    @auction.command(case_insensitive=True)
```

Comment:
case_insensitive *does not go here*
case_insensitive is passed into *group* commands or the main bot class, its not set per command

Reply:
Same goes for all the other locations you added case_insensitive

Reply:
wait, oh, my bad

Reply:
Fixed, thank you